src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,29 @@
+.block {
+  display: flex;
+  width: 166px;
```

Comment:
Same, no need to hardcode it, just add padding 32px 16px 16px to whole your card